runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -2044,6 +2044,7 @@ static void verifyDoFnSupported(DoFn<?, ?> fn, boolean streaming) {
               DataflowRunner.class.getSimpleName(), MapState.class.getSimpleName()));
     }
     if (DoFnSignatures.usesOrderedListState(fn)) {
+      // https://issues.apache.org/jira/browse/BEAM-10947
```

Comment:
nit: better to add a `TODO`.

Reply:
I followed the style of the others in this file. Do we have tooling for tracking TODOs differently?